packages/common-all/src/types/configs/commands/lookup.ts

```diff
@@ -38,7 +46,8 @@ export function genDefaultLookupConfig(): LookupConfig {
   return {
     note: {
       selectionMode: LookupSelectionModeEnum.extract,
-      confirmVaultOnCreate: false,
+      confirmVaultOnCreate: true,
+      selectVaultModeOnCreate: LookupSelectVaultModeOnCreateEnum.smart,
```

Comment:
I got confused with the wording because `selectVaultModeOnCreate` read like a boolean property but it's actually describing what the vault selection mode is.

Can we follow the convention in [[Config|dendron://dendron.docs/dev.process.code.config]]:

```
1. Use bare nouns only for configuration keys with values that are direct objects / literals.
    - A list of vaults, for example, should be in a key simply called `vaults`.
```

Maybe something like `vaultModeOnCreate` similar to how `selectionMode` is named.

Reply:
Good call, renaming.

Reply:
on the naming part, one more comment. since we aren't changing the *type of the vault* but how we are selecting vaults, can we use `vaultSelectionModeOnCreate` to be super explicit?

Reply:
That makes more sense. It's always good to be as explicit as possible.